### ISSUE TITLE
Add in JWT functionality

### ISSUE THREAD

**Issue by domharrington** [readmeio/api-explorer#49]

**Comment by domharrington** [comment]
https://github.com/readmeio/api-explorer/search?utf8=%E2%9C%93&q=replaceVars&type=

**Comment by domharrington** [comment]
@uppal101 You may not have enough information to do this one, but if you're looking for stuff maybe ask @mjcuva to show you this and take a stab at it.

**Comment by domharrington** [comment]
Latest replace vars code is here: https://github.com/readmeio/api-explorer/blob/098ae594d28b74b0aaeef307689122d8e6a17889/packages/api-explorer/src/lib/replace-vars.js

**Comment by domharrington** [comment]
Angular directive and templates are here: https://github.com/readmeio/api-explorer/tree/master/legacy-stuff/variables

**Comment by mjcuva** [comment]
## Need to support

- `<<apiKey>>` variable format
- `<<glossary:term>>` glossary terms
- User object without keys
- User object with keys as array

---
```
{
  name: 'marc',
  email: 'marc@readme.io',
  apiKey: '123',
  version: 1,
}
```

```
{
  name: 'marc',
  email: 'marc@readme.io',
  keys: [{
    apiKey: '123',
    name: 'project 1',
  }, {
    apikey: '456',
    name: 'project 2',
  }
  version: 1,
}
```

**Comment by mjcuva** [comment]
## Do Not Need to Support
 
- <<keys:api_key>> api_key format for apiKey
- <<keys:projectKey>> keys: format for nested objects
- User object with keys as object

---
```
{
  name: 'marc',
  email: 'marc@readme.io',
  keys: {
    apiKey: '123',
    name: 'project 1',
  }
  version: 1,
}
```

**Comment by domharrington** [comment]
Closed in #125
